### PR TITLE
Simplify setup

### PR DESCRIPTION
**Pull request by Alan-Jowett**
## Description

Installing all the dependencies required to build eBPF-For-Windows is difficult and error-prone. This PR adds a simplified option to install all the required dependencies.

## Testing

Manual testing.

## Documentation

No.

### CODE REVIEW

File: scripts/Setup-DevEnv.ps1 [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Invoke-WebRequest 'https://community.chocolatey.org/install.ps1' -OutFile $env:TEMP\install_choco.ps1
+if ((get-filehash $env:TEMP\install_choco.ps1).Hash -ne '6B2C4EF29B871090B758E403AEE3EFAF9018B21F90FFA03CD4E0C27506331F01') { throw "Wrong file hash for Chocolatey installer"}
+&"$env:TEMP\install_choco.ps1"
+choco install git --version 2.36.0 -y
+choco install visualstudio2019community --version 16.11.12.0 -y
+choco install visualstudio2019-workload-nativedesktop --version 1.0.1 -y
+choco install visualstudio2019buildtools --version 16.11.12.0 -y
+choco install windowsdriverkit10 --version 10.0.19041.685 -y
+choco install windowsdriverkit11 --version 10.0.22000.1 -y
+choco install llvm --version 13.0.1 -y
```
Comment (dthaler) [inline scripts/Setup-DevEnv.ps1:13]: This line contradicts line 13 of GettingStated.md which says 10 not 13.  Both files should say the same thing.

**Review by dthaler — APPROVED** [review]

File: scripts/Setup-DevEnv.ps1 [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Invoke-WebRequest 'https://community.chocolatey.org/install.ps1' -OutFile $env:TEMP\install_choco.ps1
+if ((get-filehash $env:TEMP\install_choco.ps1).Hash -ne '6B2C4EF29B871090B758E403AEE3EFAF9018B21F90FFA03CD4E0C27506331F01') { throw "Wrong file hash for Chocolatey installer"}
+&"$env:TEMP\install_choco.ps1"
+choco install git --version 2.36.0 -y
+choco install visualstudio2019community --version 16.11.12.0 -y
+choco install visualstudio2019-workload-nativedesktop --version 1.0.1 -y
+choco install visualstudio2019buildtools --version 16.11.12.0 -y
+choco install windowsdriverkit10 --version 10.0.19041.685 -y
+choco install windowsdriverkit11 --version 10.0.22000.1 -y
+choco install llvm --version 10.0.0 -y
```
Comment (shankarseal) [inline scripts/Setup-DevEnv.ps1:13]: We need 13 though, dont we? Otherwise the bpf2c tests would fail.

File: scripts/Setup-DevEnv.ps1 [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+choco install nuget.commandline --version 6.1.0 -y
```
Comment (shankarseal) [inline scripts/Setup-DevEnv.ps1:14]: I am not sure if all the installers would update the PATH env viariable.

File: scripts/Setup-DevEnv.ps1 (hunk 2 above)
Comment (dthaler) [reply scripts/Setup-DevEnv.ps1:13]: The demo tests fail with 13 according to Anurag, so we need 10 for now.  Moving to 13 is part of issue #962

File: scripts/Setup-DevEnv.ps1 (hunk 3 above)
Comment (Alan-Jowett) [reply scripts/Setup-DevEnv.ps1:14]: The chocolatey installer appears to set the path.

**Review by shankarseal — APPROVED** [review]

**Pull request merged**